web/delete_user.go

```diff
@@ -1,12 +1,13 @@
 package web
 
 import (
+	"database/sql"
 	"github.com/rohanthewiz/element"
 	"github.com/rohanthewiz/rweb"
 	"secure_journal/login"
 )
 
-func DeleteRouter(s *rweb.Server) {
+func DeleteRouter(s *rweb.Server, db *sql.DB) {
```

Comment:
Since these are basic routes and handlers, not the router itself, let's change the naming pattern to xxRoutes.